hudi-flink/src/main/java/org/apache/hudi/sink/compact/FlinkCompactionConfig.java

```diff
@@ -89,12 +89,23 @@
   @Parameter(names = {"--compaction-tasks"}, description = "Parallelism of tasks that do actual compaction, default is -1", required = false)
   public Integer compactionTasks = -1;
 
+  @Parameter(names = {"--schedule", "-sc"}, description = "Schedule the compaction plan, there is risk to lost data when this option turns on,\n"
```

Comment:
Not recommended. Schedule the compaction plan in this job. There is a risk of losing data when scheduling compaction outside the writer job. Scheduling compaction in the writer job and only let this job do the compaction execution is recommended. Default is false.